src/misc/crypt/crypt_ltc_mp_descriptor.c

```diff
@@ -8,9 +8,20 @@
  */
 #include "tomcrypt_private.h"
 
+
 /* Initialize ltc_mp to nulls, to force allocation on all platforms, including macOS. */
-ltc_math_descriptor ltc_mp = { 0 };
+/* ltc_math_descriptor ltc_mp = { 0 }; */
```

Comment:
this is the correct solution according to the C standard and if your compiler is broken we can't help you